integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -39,7 +39,7 @@
  * Utility methods for Alluxio-FUSE.
  */
 @ThreadSafe
-public final class AlluxioFuseUtils {
+public class AlluxioFuseUtils {
```

Comment:
Do you want to extend `AlluxioFuseUtils`?

Reply:
Yeah, I have a project dependent alluxio-fuse, my Class want to extends `AlluxioFuseUtils `

Reply:
It's not great to extend an utility class, so they are often made `final`. and only have static methods. I think you are not inheriting anything when you extend `AlluxioFuseUtils` right? I would actually recommend just having a separate class to keep the extra util methods.

Everything else looks good to me :)

Reply:
@jiacheliu3 Thank you for your reply, It make sense to keep final